client/me/help/help-contact/index.jsx

```diff
@@ -177,6 +183,16 @@ const HelpContact = React.createClass( {
 		this.clearSavedContactForm();
 	},
 
+	prepareDirectlyWidget: function() {
+		if (
+			this.hasDataToDetermineVariation() &&
+			this.getSupportVariation() === SUPPORT_DIRECTLY &&
+			this.props.isDirectlyUninitialized
+		) {
+			this.props.initializeDirectly();
```

Comment:
It is necessary — this isn't on production yet. It's protecting the `prepareDirectlyWidget` call at a distance because `this.getSupportVariation() === SUPPORT_DIRECTLY` will always be `false` unless the flag is enabled.